src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -235,13 +236,10 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         return queriedExtendedStates
     }
 
-
     @Override
     ExecutionResult executeKillJobs(List<BEJobID> jobIDs) {
-        StringBuilder killJobsCommand = new StringBuilder(PBS_COMMAND_DELETE_JOBS)
-        killJobsCommand << " " << jobIDs*.id.join(" ")
-        logger.always(killJobsCommand.toString())
-        return executionService.execute(killJobsCommand.toString(), false)
+        String command = "qdel ${jobIDs*.id.join(" ")}"
```

Comment:
I thought, we'd check, if jobIDs is valid? Do not execute the command, if the the list is empty...

Reply:
Yes, you're right. The methods also have a clear name so it is ok, if they fail.